lib/pool.js

```diff
@@ -27,21 +27,19 @@ function _addBackoff(factory) {
 
   factory.create = function __run() {
 
-    return originalCreateMethod().catch((e) =>
+    return originalCreateMethod().catch(function ___handleErrorInCreate(e) {
 
-      new Bluebird(function ___backoffRunResolver(resolve, reject) {
-        const wait = backoff.next()
+      const wait = backoff.next()
```

Comment:
Then I don't get the point of using back off here where you can just give it a hardcoded value.